react/SlideToggle/SlideToggle.less

```diff
@@ -0,0 +1,82 @@
+@import (reference) "~seek-style-guide/theme";
+
+.root {
+  display: inline-flex;
+  line-height: @grid-row-height * 3;
+  height: @grid-row-height * 3;
```

Comment:
This element is only 3 grid rows high. Should it have a touchable area of 5 to help on mobile? Doesn't need to actually be that big IMO though.